mantidimaging/core/utility/unit_conversion.py

```diff
@@ -42,7 +42,11 @@ def set_data_to_convert(self, data_to_convert: np.ndarray) -> None:
 
     def check_data(self) -> None:
         try:
-            self.velocity = self.target_to_camera_dist / (self.tof_data_to_convert + self.data_offset)
+            tof_with_offset = self.tof_data_to_convert + self.data_offset
+            self.velocity = np.divide(self.target_to_camera_dist,
+                                      tof_with_offset,
+                                      out=np.full_like(tof_with_offset, np.inf, dtype=np.float64),
+                                      where=tof_with_offset != 0)
```

Comment:
Is there a case where time is zero? I wonder if we should be raising an error for that case as it probably means something has gone wrong.